presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -193,6 +193,9 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             return value;
         }
+        else if (type.toString().equals("ObjectId")) { // So there's no circular dependency (presto-mongodb -> presto-tests -> presto-mongodb)
```

Comment:
@martint Yeah I was afraid of that. Without it, the test client will throw the AssertionError when testing queries that include ObjectIds. How would you recommend this get refactored?